Source/Core/DolphinWX/WxUtils.h

```diff
@@ -32,6 +33,32 @@ wxBitmap CreateDisabledButtonBitmap(const wxBitmap& original);
 // Helper function to add a button to a toolbar
 void AddToolbarButton(wxToolBar* toolbar, int toolID, const wxString& label, const wxBitmap& bitmap, const wxString& shortHelp);
 
+// Convert a coordinate from 96DPI "device independent pixel" coordinates (aka "CSS Pixels")
+// to 'real' screen pixels. This is more convenient than wxDLG_UNIT for things like drawing,
+// borders and element spacing.
+// NOTE: This function exists natively in wxWidgets 3.1.0 as wxWindowBase::FromDIP()
```

Comment:
I think it would be beneficial to just update our currently supported version of wx in our externals.